kasa/discover.py

```diff
@@ -239,9 +245,13 @@ async def discover(
         :param target: The target address where to send the broadcast discovery
          queries if multi-homing (e.g. 192.168.xxx.255).
         :param on_discovered: coroutine to execute on discovery
-        :param timeout: How long to wait for responses, defaults to 5
+        :param discovery_timeout: How long to wait for responses, defaults to 5
```

Comment:
```suggestion
        :param discovery_timeout: seconds to wait for responses, defaults to 5
```

Reply:
Done and elsewhere for other timeout comments